zoo/repos/zoo_yml.py

```diff
@@ -100,4 +112,12 @@ def generate(service: Service) -> str:
         }
         result["environments"].append(environ)
 
+    for link in service.links.all():
+        link = {
+            "name": link.name,
+            "url": link.url,
+            "icon": link.icon,
+        }
+        result["links"].append(link)
```

Comment:
nitpick: i personally like it more like this 😄️ 
```suggestion 
        result["links"].append(
	          {
	              "name": link.name,
	              "url": link.url,
	              "icon": link.icon,
	          }
        )
```